sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -191,171 +185,45 @@ export interface AttestTpmOptions extends AttestationClientOperationOptions {
 export type CertificateModification = string;
 
 // @public
-export interface CloudError {
-    error?: CloudErrorBody;
-}
-
-// @public
-export interface CloudErrorBody {
-    code?: string;
-    message?: string;
-}
-
-// @public
-export type DataType = string;
-
-// @public (undocumented)
-export class GeneratedClient extends GeneratedClientContext {
-    // Warning: (ae-forgotten-export) The symbol "GeneratedClientOptionalParams" needs to be exported by the entry point index.d.ts
-    constructor(credentials: coreAuth.TokenCredential, instanceUrl: string, options?: GeneratedClientOptionalParams);
-    // Warning: (ae-forgotten-export) The symbol "Attestation" needs to be exported by the entry point index.d.ts
-    //
-    // (undocumented)
-    attestation: Attestation;
-    // Warning: (ae-forgotten-export) The symbol "MetadataConfiguration" needs to be exported by the entry point index.d.ts
-    //
-    // (undocumented)
-    metadataConfiguration: MetadataConfiguration;
-    // Warning: (ae-forgotten-export) The symbol "Policy" needs to be exported by the entry point index.d.ts
-    //
-    // (undocumented)
-    policy: Policy;
-    // Warning: (ae-forgotten-export) The symbol "PolicyCertificates" needs to be exported by the entry point index.d.ts
-    //
-    // (undocumented)
-    policyCertificates: PolicyCertificates_2;
-    // Warning: (ae-forgotten-export) The symbol "SigningCertificates" needs to be exported by the entry point index.d.ts
-    //
-    // (undocumented)
-    signingCertificates: SigningCertificates;
-}
-
-// @public (undocumented)
-export class GeneratedClientContext extends coreClient.ServiceClient {
-    constructor(credentials: coreAuth.TokenCredential, instanceUrl: string, options?: GeneratedClientOptionalParams);
-    // (undocumented)
-    apiVersion: string;
-    // (undocumented)
-    instanceUrl: string;
-}
-
-// @public (undocumented)
-export interface JsonWebKey {
-    alg?: string;
-    crv?: string;
-    d?: string;
-    dp?: string;
-    dq?: string;
-    e?: string;
-    k?: string;
-    kid?: string;
-    kty: string;
-    n?: string;
-    p?: string;
-    q?: string;
-    qi?: string;
-    use?: string;
-    x?: string;
-    x5C?: string[];
-    y?: string;
-}
-
-// @public (undocumented)
-export interface JsonWebKeySet {
-    keys?: JsonWebKey[];
-}
-
-// @public
-export enum KnownAttestationType {
+export const enum KnownAttestationType {
     OpenEnclave = "OpenEnclave",
     SgxEnclave = "SgxEnclave",
     Tpm = "Tpm"
 }
 
 // @public
-export enum KnownCertificateModification {
+export const enum KnownCertificateModification {
```

Comment:
this should be reverted back to not be a const. This is still an issue in autorest.

Reply:
Confused - how do I revert it back, this is a machine generated file.

Reply:
It has to be edited by hand for now. @sarangan12 do we have an ETA for when the fix in autorest will be released?

Reply:
Ah, I think I figured it out. The problem is that we're now exporting the autorest type definitions instead of the hand written versions. That way when the swagger is updated (which may happen in a month or so), we will automatically pick up those changes when we adopt the new API version.

Reply:
Btw, the hand edit suggestion results in a build error, so that's a non-starter.

I'm open to other suggestions, but I'd REALLY rather not create a hand-written copy of the generated types.

Reply:
I am working on the fix right now. The PR will be created today. I am expecting a release on Tuesday.